keyvalue/rust/src/keyvalue.rs

```diff
@@ -1099,7 +1196,6 @@ impl<T: Transport + std::marker::Sync + std::marker::Send> KeyValue for KeyValue
             .map_err(|e| RpcError::Deser(format!("'{}': I32", e)))?;
         Ok(value)
     }
-
```

Comment:
Not sure what's up with all of these whitespace changes...just ran `make`.  Should I remove them?